common/actions/wallet/actionTypes.ts

```diff
@@ -17,6 +17,10 @@ export interface UnlockMnemonicAction {
   payload: MnemonicUnlockParams;
 }
 
+export interface UnlockWeb3Action {
+  type: 'WALLET_UNLOCK_WEB3';
```

Comment:
Not sure how it slipped but all the `type` values in this file should be using the `enum` from `constants.ts`

Reply:
Converted `constants.ts` to the `enum` format and switched the wallet action files over.